about.html

```diff
@@ -83,6 +82,26 @@ <h2 class="venue-subtitle-text"></h2>
         </li>
       </ul>
     </section>
+    <!-- Section 4: Contact & Inquiries-->
+      <h1>Contact &amp; Inquiries</h1>
+      <p>
+        Someone from the CPOSC team will get back to you as soon as possible.
+      </p>
+      <form action="">
+        Name:<br>
+        <input class="small-input-box" type="text" name="fullname" value=""><br>
+        Email:<br>
+        <input class="small-input-box" type="text" name="email" value=""><br>
+        Phone:<br>
+        <input class="small-input-box" type="text" name="phone" value=""><br>
+        Message:<br>
+        <input class="large-input-box" type="text" name="message" value=""><br>
 
+      <section class="submit-background">
+        <input class="submit-button" type="button" value="SUBMIT">
+      </section>
+      </form>
+
```

Comment:
BIG PROBLEM::::: theres an extra space

Reply:
There we go I commented and fixed that darn space XD